### PR TITLE
Migrated meeting sidepandel nodejs sample to use live-share-sdk

### PR DESCRIPTION
**Pull request by Prithvi-MSFT**

### CODE REVIEW

**Review by AbhijeetThakur-msft — APPROVED** [review]

File: samples/meetings-sidepanel/nodejs/ClientApp/package.json [hunk 1]
```diff
@@ -0,0 +1,67 @@
+{
+  "name": "live-coding-sample",
+  "version": "0.3.1",
+  "description": "This sample demonstrates live coding using live share sdk.",
```
Comment (ChetanSharma-msft) [inline samples/meetings-sidepanel/nodejs/ClientApp/package.json:4]: Is this live coding sample?

File: samples/meetings-sidepanel/nodejs/ClientApp/package.json [hunk 2]
```diff
@@ -0,0 +1,67 @@
+{
+  "name": "live-coding-sample",
```
Comment (ChetanSharma-msft) [inline samples/meetings-sidepanel/nodejs/ClientApp/package.json:2]: Is this live coding sample?

File: samples/meetings-sidepanel/nodejs/ClientApp/public/index.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Web site created using create-react-app"
```
Comment (ChetanSharma-msft) [inline samples/meetings-sidepanel/nodejs/ClientApp/public/index.html:10]: Change the content

File: samples/meetings-sidepanel/nodejs/README.md [hunk 4]
```diff
@@ -30,7 +30,6 @@ This sample illustrates how to implement [Side Panel](https://docs.microsoft.com
     # determine node version
     node --version
     ```
-- [socket.io](https://www.npmjs.com/package/socket.io) to update data in real time.
       
  - [Ngrok](https://ngrok.com/download) (Only for devbox testing) Latest (any other tunneling      software       can also be used)
```
Comment (ChetanSharma-msft) [inline samples/meetings-sidepanel/nodejs/README.md:34]: Check for spacing

File: samples/meetings-sidepanel/nodejs/server/index.js [hunk 5]
```diff
@@ -55,7 +55,7 @@ const bot = new SidePanelBot();
 // Create HTTP server.
 const server = restify.createServer(
     );
-const io = socketio.listen(server.server);
+//const io = socketio.listen(server.server);
```
Comment (ChetanSharma-msft) [inline samples/meetings-sidepanel/nodejs/server/index.js:58]: Remove it if not needed.

File: samples/meetings-sidepanel/nodejs/teamsAppManifest/manifest.json [hunk 6]
```diff
@@ -2,14 +2,17 @@
   "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.12/MicrosoftTeams.schema.json",
   "manifestVersion": "1.12",
   "version": "1.0.0",
-  "id": "<<YOUR-MICROSOFT-APP-ID>>",
-  "packageName": "com.microsoft.teams.sample.meetingtoken",
-
+  "id": "<<Manifest-id>>",
+  "packageName": "com.teams.livecodinginterview",
```
Comment (ChetanSharma-msft) [inline samples/meetings-sidepanel/nodejs/teamsAppManifest/manifest.json:6]: Is this live coding sample?

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
Reviewed and provided my comments

File: samples/meetings-sidepanel/nodejs/README.md [hunk 7]
```diff
@@ -30,14 +30,14 @@ This sample illustrates how to implement [Side Panel](https://docs.microsoft.com
...
-    ```bash
 
+    ```bash
      # run ngrok locally
     ngrok http -host-header=localhost 3978
```
Comment (ChetanSharma-msft) [inline samples/meetings-sidepanel/nodejs/README.md:38]: Please verify the port numbers and project folder path once as you have mentioned two ports like 3978 & 3001

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine, approving the PR.

**Review by Wajeed-msft — APPROVED** [review]

**Pull request merged**